### ISSUE TITLE
Beta feedback - RahulHP

### ISSUE THREAD

**Issue by RahulHP** [Brok-Bucholtz/P4-Beta#1]
- **Text to Word Ids** 
Clarify this point "The sentences have already been converted to lowercase." in the "Text to Word Ids" section
- **Encoding**
Clarify that `LSTMCell()` is to be used (based on training review) and that dropout is optional (based on hints in project review page) in the "Encoding" section. "Use tf.contrib.rnn.LSTMCell() as the building block of the network. Optionally, implement dropout with keep probability keep_prob."
- **Positional vs keyword arguments**
Given the huge number of arguments every function has in Tensorflow, I'd like to add a note in the beginning promoting the usage of positional arguments instead of keyword arguments for clarity and to avoid mistakes.
